packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -6202,27 +6197,31 @@ export default class Meeting extends StatelessWebexPlugin {
    * Functionality for when a share video is ended.
    * @private
    * @memberof Meeting
-   * @param {MediaStream} localShare
    * @returns {undefined}
    */
   private handleShareVideoTrackEnded = async () => {
-    if (this.wirelessShare) {
-      this.leave({reason: MEETING_REMOVED_REASON.USER_ENDED_SHARE_STREAMS});
-    } else {
-      try {
-        await this.unpublishTracks([this.mediaProperties.shareVideoTrack]);
-      } catch (error) {
-        LoggerProxy.logger.log(
-          'Meeting:index#handleShareVideoTrackEnded --> Error stopping share: ',
-          error
-        );
-      }
+    try {
+      await this.unpublishTracks([this.mediaProperties.shareVideoTrack]);
+    } catch (error) {
+      LoggerProxy.logger.log(
+        'Meeting:index#handleShareVideoTrackEnded --> Error stopping share: ',
+        error
+      );
     }
     this.triggerStoppedSharing();
   };
 
+  /**
+   * Emits meeting:stoppedSharingLocal
+   * @private
+   * @returns {undefined}
+   * @memberof Meeting
+   */
   private triggerStoppedSharing = () => {
     if (!this.mediaProperties.shareAudioTrack && !this.mediaProperties.shareVideoTrack) {
+      if (this.wirelessShare) {
```

Comment:
if we do this handling of wirelessShare now here, then the name and comments of the `triggerStoppedSharing()` function need to be updated, because they don't match what the function is doing anymore.
Also previously we would only call `unpublishTracks()` in the `handleShareTrackEnded()` if `this.wirelessShare` was false, now we always call it and then do the leave, this may cause a problem especially for transcoded meetings because unpublish will result in a new SDP exchange and we will leave before that exchange is finished which can cause some undesired error metrics for the backend teams.

Reply:
moved it back to the handleShareTrackEnded methods and added a check for both share tracks to have ended before leaving